net.sf.nootka.json

```diff
@@ -0,0 +1,123 @@
+{
+  "app-id": "net.sf.nootka",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "nootka",
+  "branch": "1.4.5",
+
+  "rename-appdata-file": "nootka.appdata.xml",
+  "rename-desktop-file": "nootka.desktop",
+  "rename-icon": "nootka",
+
+  "finish-args":[
+      "--device=dri",
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=wayland",
+      "--share=network",
+      "--filesystem=host",
+      "--socket=pulseaudio"
+  ],
+
+  "modules": [
+    {
+      "name": "fftw",
+      "rm-configure": true,
+      "config-opts": [
+        "--enable-shared", "--enable-float"
+      ],
+      "cleanup": [
+          "/bin",
+          "/share/info",
+          "/include",
+          "/share/man",
+          "/lib/cmake",
+          "/lib/debug",
+          "/lib/pkgconfig",
+          "*.a",
+          "*.la"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://www.fftw.org/fftw-3.3.7.tar.gz",
+          "sha256": "3b609b7feba5230e8f6dd8d245ddbefac324c5a6ae4186947670d9ac2cd25573"
+        },
+        {
+          "type": "script",
+          "commands": [
+            "autoreconf -fiv"
+          ],
+          "dest-filename": "autogen.sh"
+        }
+      ]
+    },
+
+    {
+      "name": "soundtouch",
+      "config-opts": [
+        "--enable-shared", "--disable-static"
+      ],
+      "sources": [
+        {
+            "type": "archive",
+            "url": "http://www.surina.net/soundtouch/soundtouch-2.0.0.tar.gz",
+            "sha256": "d224f7d1421b5f8e74a74c85741345bd9802618a40ae30ce5b427a5705c89d25"
+        }
+      ],
+      "cleanup": [
+          "/bin",
+          "/include",
+          "/lib/pkgconfig",
+          "/lib/debug",
+          "/share/aclocal",
+          "/share/doc"
+      ]
+    },
+
+    {
+      "name": "appdata",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "file",
+          "path": "nootka.appdata.xml"
+        }
+      ],
+      "build-commands": [
+        "mkdir -p /app/share/appdata",
+        "cp nootka.appdata.xml /app/share/appdata"
+      ]
+    },
+
+    {
+      "name": "nootka",
+      "buildsystem": "cmake-ninja",
+      "builddir": true,
+      "config-opts": [
+          "-DFFTW3_INCLUDE_DIRS=/app/include",
+          "-DFFTW3_LIBRARIES=/app/lib/libfftw3f.so"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/nootka/files/Nootka-1.4.5/nootka-1.4.5-source.tar.bz2",
+          "sha256": "210371ffeb8f018564e1d07d935dee7f310316c67359ac40569cc74f3c3a5e16"
+        }
+      ],
+      "post-install": [
+        "mkdir -p /app/share/icons/hicolor/512x512/apps",
+        "mkdir -p /app/share/icons/hicolor/64x64/apps",
+        "cp /app/share/nootka/picts/nootka.png /app/share/icons/hicolor/64x64/apps",
```

Comment:
I haven't checked yet (I will as it builds) but is this also putting a 128x128px icon in the correct path? It needs that for Software stores.

Reply:
Added 128x128 icon. In next stable release I will try to make those icons 'native'. So far 'share/pixmaps' was used for that.